src/Types/Input/EnumInputType.hack

```diff
@@ -32,13 +28,13 @@ abstract class EnumInputType extends NamedInputType {
         $value = $node->getRawValue();
         $enum = static::HACK_ENUM;
         if (!$node is Value\EnumLiteral || !$value is string || !C\contains_key($enum::getValues(), $value)) {
-            throw new GraphQL\UserFacingError(
-                'Expected a valid value for %s, got %s',
-                static::NAME,
-                \get_class($node)
-            );
+            throw new GraphQL\UserFacingError('Expected a valid value for %s, got %s', static::NAME, \get_class($node));
         }
         return $enum::getValues()[$value];
     }
 
+    public function getTypeKind(): GraphQL\Introspection\__TypeKind {
+        return GraphQL\Introspection\__TypeKind::ENUM;
```

Comment:
Should this just be a class constant exposes via a method?

Reply:
Yea... I thought I did that at first but there was some reason I wanted a method. I can't remember.

Reply:
If we want to put `<<GraphQL\Field>>` on these (either here or in the parent interface) -- which I think is worth trying, so we can avoid all those wrapper classes -- then we'll need these to be methods, since `<<GraphQL\Field>>` doesn't work on constants.